shared/common-adapters/input.native.js

```diff
@@ -1,13 +1,11 @@
-/* @flow */
-
-import React, {Component} from 'react'
-import {TextInput} from 'react-native'
-import {globalColors, globalStyles} from '../styles/style-guide'
+// @flow
+import Box from './box'
 import Platform, {OS} from '../constants/platform'
+import React, {Component} from 'react'
 import Text from './text.native'
-import Box from './box'
-
 import type {Props} from './input'
+import {TextInput} from 'react-native'
+import {globalColors, globalStyles} from '../styles/style-guide'
```

Comment:
It's literally just alphabetized. Simplest for now. There at eslint plugins but they all have problems